src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-baseline-alert-conditions.mdx

```diff
@@ -59,6 +59,20 @@ Here are some tips for setting baseline thresholds:
 * For NRQL alerts, see the [allowed types of NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#threshold-types).
 * If the alert condition applies to multiple apps, you can select a choice from the dropdown above the chart to use different metrics. (Not applicable for NRQL alert conditions.)
 
+## Faceted baseline conditions [#baseline-facets]
+
+![A screenshot of an example baseline condition with facets.](./images/faceted-baseline-condition-query.png "An example of a baseline condition with facets.")
```

Comment:
This screenshot is a bit blurry and covers a lot of ground. Could you just focus on one part of the screen and make it bigger? I'm not sure what's important in this image. For example, could you just show the left half of the image and still get the point across?